cmd/demo.yaml

```diff
@@ -82,10 +108,36 @@ chargers:
       source: js
       vm: shared
       script: |
+        console.log("Outer Set!!!");
         set();
         var lp = state.loadpoints[0];
         lp.enabled = val;
-        if (lp.enabled) lp.chargepower = lp.maxcurrent * 230 * lp.phases; else lp.chargepower = 0;
+        if (lp.enabled) ret = lp.maxcurrent * 230 * lp.phases; else ret = 0;
+        ret;
+      transform:
```

Comment:
Das find ich wahnsinnig unglücklich. Die Transformation passiert ja genau in dem getter/setter, also muss die Config auch da hin. Das ist für Endanwender sonst nicht beherrschbar.